winlogbeat/eventlog/wineventlog.go

```diff
@@ -332,7 +332,8 @@ func (l *winEventLog) Read() ([]Record, error) {
 		if r.Message == "" {
 			r.Message, err = l.message(h)
 			if err != nil {
-				logp.Err("%s error salvaging message: %v", l.logPrefix, err)
+				logp.Warn("%s error salvaging message (event id=%d qualifier=%d created at %s will be included without a message): %v",
```

Comment:
I think it would be useful to have the Provider.Name associated to the event ID.